src/Magento/FunctionalTestingFramework/Util/Logger/MftfLogger.php

```diff
@@ -28,4 +28,21 @@ public function deprecation($message, array $context = [])
         }
         parent::warning($message, $context);
     }
+
+    /**
+     * Prints a critical failure, as well as adds a log at the CRITICAL level.
+     *
+     * @param  string $message The log message.
+     * @param  array  $context The log context.
+     * @return void
+     */
+    public function buildFailure($message, array $context = [])
```

Comment:
Rename to just `critical` to follow pattern set by `deprecation` function above.